util/src/io/service.rs

```diff
@@ -231,8 +231,8 @@ impl<Message> Handler for IoManager<Message> where Message: Send + Clone + Sync
 	fn notify(&mut self, event_loop: &mut EventLoop<Self>, msg: Self::Message) {
 		match msg {
 			IoMessage::Shutdown => {
-				self.workers.clear();
 				event_loop.shutdown();
```

Comment:
If event loop is destroyed while workers are active they might panic trying to send notification to the event loop. Any particular reason to remove this?
If there are unhandled events left in the event loop that's fine cause we just ignore them. We don't have to process anything after `Shutdown` event.